charts/mariadb/templates/mariadb-deployment.yaml

```diff
@@ -30,7 +30,27 @@ spec:
           name: mariadb
           ports:
             - containerPort: 3306
-          resources: {}
+          {{- if .Values.resources }}
+          resources:
+            {{- if .Values.resources.requests }}
+            requests:
+              {{- if .Values.resources.requests.memory }}
+              memory: {{  .Values.resources.requests.memory }}
```

Comment:
it makes sense to add specific limits per service, like .Values.resources.requests.memory should be changed to .Values.resources.mariadb.memory, I don't think that it's a good idea to have same definitions for keycloak, mariadb and mongo

Reply:
as agreed, it can be overriden with chart name , so it's okay (as these external services are managed as subcharts)